plutus-pab/src/Cardano/Node/Types.hs

```diff
@@ -75,6 +76,37 @@ import qualified Wallet.Emulator.MultiAgent     as MultiAgent
 
 import           Ledger.Fee                     (FeeConfig)
 import           Plutus.PAB.Arbitrary           ()
+-- =======
```

Comment:
merge artifacts?

Reply:
Yeah. I reviewed the code before finding out there were conflicts.